src/docs/admin/references/properties/scheduler.properties

```diff
@@ -10,24 +10,24 @@ pa.scheduler.home=.
 # you can use an absolute location or a relative location from scheduler home
 # pa.hsqldb.location=data/db
 
-# Scheduler REST url. If not defined, it is set automatically when starting the server.
+# Scheduler rest url. If not defined, it is set automatically when starting the server.
```

Comment:
REST => rest. Here as well if you want https://github.com/ow2-proactive/documentation/pull/729/files?short_path=ad0eb21#diff-607f3f73524204d4e30d7dfaa6bfc75152d69d7867d866dfc3a96d011644ef28L296

Reply:
same remarks